.github/actions/spelling/patterns.txt

```diff
@@ -60,7 +69,7 @@ REQUIRE\(RestHelper::GetRestAPIBaseUri\(".*"\) == L".*"
 # Automatically suggested patterns
 # hit-count: 407 file-count: 78
 # IServiceProvider
-\bI(?=(?:[A-Z][a-z]{2,})+\b)
+\b[IT](?=(?:[A-Z][a-z]{2,})+(?:[A-Z]|\b))
```

Comment:
There were a handful of `T...` items (flagged earlier), and it seemed worthwhile to add `T` here. There are also a couple of other letters which each have a couple of items, but it didn't really seem worthwhile to include a blanket -- it is worth noting that the pattern could be widened to cover them if that's desired.

Reply:
Does this still check the spelling of the word after the `T`? Or would it allow stuff like `TMispeled`?

Reply:
Yes, the `(?=...)` is a [zero-width lookahead](https://www.regular-expressions.info/lookaround.html)